composer.json

```diff
@@ -4,22 +4,16 @@
     "keywords": ["symfony", "health", "check"],
     "type": "symfony-bundle",
     "license": "GPL-2.0-only",
```

Comment:
Shouldn't we update this parameter as well?

Reply:
very good catch ! I forgot it.

indeed, otherwise packagist and SF flex bot would never have seen the licence update.

fixed :)